src/lib.rs

```diff
@@ -27,7 +27,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 #![warn(unreachable_pub)]
-#![deny(warnings, missing_docs, clippy::as_conversions)]
+#![deny(missing_docs, clippy::as_conversions)]
```

Comment:
What's up with removing `deny(warnings)`? Unintentional? In either case, probably should be separated from the commit re-arranging intermediate cert representation.

Reply:
No, it was intentional. While developing, I often have some stages where I don't care about warnings (for example, for unused imports). We have CI checks that deny warnings anyway, which I think is sufficient.